scripts/min_glibc_version.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# This scripts determines the minimum GLIBC version required for a dynamic library.
```

Comment:
I'm a bit hesitant about re-introducing bash scripts again.
The new Rust infra, although more verbose, is easier to work with/refactor/modularize. And something like the logic below can be a simple `regex::new(..).captures_iter().max()` chain. Thoughts? Not blocking though if the VS Code deployment is blocked on it.

Reply:
The script is pretty trivial, as there's no control flow or variables and each operations seems simple and is documented; we always have to go through `Command` to get this output (whether it's `objdump` or this script), so this probably doesn't streamline too much.

I understand the need to keep the Bash scripts to the minimum and agree but this is such a self-contained piece of logic that's platform-specific (only makes sense on and for Linux/GNU; assumes and signals the need for `objdump`) that I think it strikes a right balance here.

I'll attempt a brief rewrite and and replace it with it if it's cleaner, what do you think?

Reply:
Converted to Rust-only code with a test in https://github.com/NomicFoundation/slang/pull/909/commits/6414218b63c10f473a9295841397abd35f2b18be

Reply:
Sounds great. Thanks!